docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -22,7 +22,7 @@ You can check them out on github:
 The first one adds extra metadata to versions about object parents at the moment of version creation.
 The second module extends CMS History UI adding control over nested objects.
 
-<img width="600" src="../../_images/snapshot-admin.png" width="600">
+![](../../_images/snapshot-admin.png)
```

Comment:
It's probably worth adding some alt test for accessibility

Reply:
haha, right. I've been spoiled by bad examples at https://github.com/silverstripe/silverstripe-framework/blob/4/docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Customise_React_Components.md

Reply:
Yep, that doesn't block merge though.